docs/tre-admins/setup-instructions/installing-base-workspace.md

```diff
@@ -5,7 +5,7 @@
 1. Run:
 
     ```cmd
-    register-bundle DIR=./templates/workspaces/base
+    /workspaces/tre> make register-bundle DIR=./templates/workspaces/base
     ```
 
     Copy the resulting payload json.
```

Comment:
```suggestion
    /workspaces/tre> make register-bundle DIR=./templates/workspaces/base BUNDLE_TYPE=workspace
    ```

    Copy the resulting JSON payload.
```

Reply:
Related to this, a change is required in the bundle registration script (currently here https://github.com/microsoft/AzureTRE/compare/develop...tompaana/docs-fixes?expand=1 - later in Tess' PR).